src/systems/physics/Physics.hh

```diff
@@ -64,6 +64,19 @@ namespace systems
 
   /// \class Physics Physics.hh ignition/gazebo/systems/Physics.hh
   /// \brief Base class for a System.
+  /// Includes optional parameter : <include_entity_names>. When set
```

Comment:
This is good. For other systems we have been listing the parameters in a bullet list, but since this is the only parameter here, I think the format is fine.